WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -61,6 +61,11 @@ + (NSArray *)routes
     [[FBRoute POST:@"/tap/:uuid"] respondWithTarget:self action:@selector(handleTap:)],
     [[FBRoute POST:@"/keys"] respondWithTarget:self action:@selector(handleKeys:)],
     [[FBRoute GET:@"/window/size"] respondWithTarget:self action:@selector(handleGetWindowSize:)],
+    
+    // For Support Swipe
+    [[FBRoute GET:@"/element/:uuid/swipeLeft"] respondWithTarget:self action:@selector(handleSwipeLeft:)],
+    [[FBRoute GET:@"/element/:uuid/swipeRight"] respondWithTarget:self action:@selector(handleSwipeRight:)],
```

Comment:
Make it one `/element/:uuid/swipe` endpoint and pass direction as parameter.

Reply:
what's uuid ?? is it element id or  device  id?